### DISCUSSION TITLE
How to link to the release version of Qt while keeping my code in debug mode?

### DISCUSSION THREAD

**Discussion by f-zl** [xmake-io/xmake#3471] [Q&A]
xmake.lua:
```lua
target("testQt")
    add_rules("qt.widgetapp")
    add_files("main.cpp")
```
command line:
```
xmake f -m debug
xmake
```
xmake will try linking to the debug version of Qt, for example, use `-lQt6Guid` instead of `-lQt6Gui`. How can I use the release version of Qt while keeping my code in debug mode?

**Comment by waruqi** [comment]
no way now.

**Reply by waruqi** [reply]
you can use other custom debug mode instead of debug mode. `mydebug`, ...

**Comment by MoonShadowIllusion** [comment]
1. copy ```bin/Qt6Gui.dll``` ```bin/Qt6Widgets.dll``` ```bin/Qt6Cored.dll``` to ```lib/```
2. add suffix to them, like ```Qt6Guid.dll``` ```Qt6Widgetsd.dll``` ```Qt6Cored.dll```
